web/html/src/components/coco-attestation/Utils.tsx

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+
+import { localizedMoment } from "utils/datetime";
+
+export type AttestationResult = {
+  id: number;
+  resultType: string;
+  resultTypeLabel: string;
+  status: string;
+  statusDescription: string;
+  description: string;
+  details: string;
+  attestationTime: Date;
+};
+
+export type AttestationReport = {
+  id: number;
+  systemId: number;
+  systemName: string;
+  status: string;
+  statusDescription: string;
+  creationTime: Date;
+  modificationTime: Date;
+  attestationTime: Date;
+  actionId: number | null;
+  actionName: string | null;
+  actionScheduledBy: string | null;
+  results: Array<AttestationResult>;
+};
+
+export function renderTime(time: Date): React.ReactNode {
```

Comment:
Maybe instead of this, we can extend `web/html/src/components/datetime/FromNow.tsx`? If we add `Date` as a valid proptype for it which should require no code changes, we should be able to drop everything besides the `null` value handling here and replace it with that component.

```tsx
export function renderTime(time: Date): React.ReactNode {
  if (time === null) {
    return t("N/A");
  }
  return <FromNow value={time} />;
}
```

Reply:
Fixed. Not sure how I missed the existing component, I even looked for it :man_facepalming: